receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -158,6 +167,14 @@ metrics:
     unit: "{transactions}/s"
     gauge:
       value_type: double
+  sqlserver.database.io.read_latency:
```

Comment:
I decided to name this `sqlserver.database.io.read_latency` instead of `sqlserver.database_io.read_latency` to try to closer match [semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-naming) for `io`.

Reply:
Updated to cumulative sum based on underlying [metric description](https://learn.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-io-virtual-file-stats-transact-sql?view=sql-server-ver16) (refer to `io_stall_read_ms`.)